csharp/roslyn-sdk/Tutorials/MakeConst/MakeConst.Vsix/source.extension.vsixmanifest

```diff
@@ -15,8 +16,7 @@
     <Asset Type="Microsoft.VisualStudio.MefComponent" d:Source="Project" d:ProjectName="MakeConst" Path="|MakeConst|"/>
     <Asset Type="Microsoft.VisualStudio.Analyzer" d:Source="Project" d:ProjectName="MakeConst" Path="|MakeConst|"/>
   </Assets>
-  <Prerequisites>
-    <Prerequisite Id="Microsoft.VisualStudio.Component.CoreEditor" Version="[15.0,16.0)" DisplayName="Visual Studio core editor" />
-    <Prerequisite Id="Microsoft.VisualStudio.Component.Roslyn.LanguageServices" Version="[15.0,16.0)" DisplayName="Roslyn Language Services" />
-  </Prerequisites>
+    <Prerequisites>
```

Comment:
Indentation is also off here. Please restore the `Roslyn.LanguageServices` component, and make them both `[15.0,17.0)`. Perhaps @jmarolf can confirm whether we can use an open-ended range here as well to prevent this from happening again in the future.

Reply:
The only change that is needed here is for the version number to be extended to include `17.0`

Reply:
Yea, somehow just removing the version ranges never worked for me, maybe bad luck ¯\_(ツ)_/¯

Now it works for me aswell
Can i leave it open ranged or is the 17.0 necessary?

Reply:
I think leaving in open is fine here. In 100 years if VS `200.0` isn't compatible we can come back and update the documentation